src/components/compress/index.tsx

```diff
@@ -352,7 +353,9 @@ export default class Compress extends Component<Props, State> {
       loading: true,
     });
 
-    this.setState({ images });
+    this.setState({ images }, () => this.base!.dispatchEvent(
+      new CustomEvent('squooshingdone', { bubbles: true }),
```

Comment:
Also somewhat of a hacky shortcut. Not sure what y’all think of this.

Reply:
Won't this fire way before squooshing is done? This `setState` is just for the loading state.

Feels like we need something else like the compression job promises.